nixos/modules/services/networking/shellhub-agent.nix

```diff
@@ -85,7 +94,6 @@ in {
         ExecStart = "${cfg.package}/bin/agent";
       };
     };
-
-    environment.systemPackages = [ cfg.package ];
```

Comment:
Is there no reason to add the package to the environment, e.g.: so the user can run the commands to check the agent status? If yes, I would keep the package on `environment.systemPackages`.

Reply:
Not currently.